specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-12-01-preview/containerInstance.json

```diff
@@ -855,6 +924,54 @@
             "modelAsString": true
           }
         }
+      },
+      "required": [
+          "name",
+          "display"
+      ]
+    },
+    "UsageListResult": {
+      "description": "The response containing the usage data",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Usage"
+          }
+        }
+      }
+    },
+    "Usage": {
+      "description": "A single usage result",
+      "type": "object",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "description": "Unit of the usage result"
+        },
+        "currentValue": {
+          "type": "integer",
+          "description": "The current usage of the resource"
+        },
+        "limit": {
+          "type": "integer",
+          "description": "The maximum permitted usage of the resource."
+        },
+        "name": {
+          "type": "object",
+          "description": "The name object of the resource",
+          "properties": {
+            "value":{
+              "type": "string",
+              "description": "The name of the resource"
+            },
+            "localizedValue": {
+              "type": "string",
+              "description": "The localized name of the resource"
+            }
+          }
```

Comment:
It looks like the properties of this model are read-only (which means generated client SDK exposes getters for them but not setters), if yes please annotate these properties with readonly e.g. attribute https://github.com/Azure/azure-rest-api-specs/blob/ca7dadf950700baa6a9fde9e78bd80d4498213f4/specification/compute/resource-manager/Microsoft.Compute/2017-12-01/compute.json#L3724

Reply:
Also UsageListResult::value